cmd/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+# Install the bash completion script in an unversioned location.
+install(FILES plugin.bash_completion.sh DESTINATION share/ignition/ignition.completion.d)
```

Comment:
Same comment as on the other PR, can we install this to `gz`? I understand this is targeting Citadel, but it would be good not to add new things to `ignition` as long as it's reasonable.

Reply:
3f84528 + b078cd1